### ISSUE TITLE
It doesn't work in ST3 on Mac OS

### ISSUE THREAD

**Issue by Arichy** [Monnoroch/ColorHighlighter#452]
I've reinstalled the ColorHighlighter plugin for 130000 times, however, it just doesn't work as if it didn't exist. 
Sublime Text 3
Mac OS 10.13.4
ColorHighlighter v8.0.9

**Comment by mattkaye** [comment]
Yup same issue. Plugin appears to do nothing. Same system config as @Arichy.

**Comment by steel-philipp** [comment]
Same issue. Got it to work super briefly by messing around with the user settings for the package. Then it just stopped. Super bummed because this looks like a really cool tool.

**Comment by jsardev** [comment]
Same here 😢

**Comment by stephenberndt** [comment]
I'm not even seeing this in package control anymore. On ST3 on a mac.

**Comment by crimerrsplay** [comment]
Same here
MacOS 10.13
Sublime Text 3 (3170)

**Comment by mobyjames** [comment]
Same

**Comment by ramilushev** [comment]
In the settings, check that the filetype you're using is included in the "file_extensions" section. I was using `.js` files and was wondering why it wasn't working. Added it and now works like a charm.

**Comment by crimerrsplay** [comment]
@ramilushev not help

**Comment by renatonmendes** [comment]
Same for me. MacOS High Sierra, ST3.

**Comment by glomotion** [comment]
Tis working for me, although the gutters are not. Weird.
MacOS 10.13.5 && SublimeText 3.1.1

**Comment by chevdor** [comment]
I confirm the issue *and* the fix.
I was trying to see the coloring in a js file.

Location the settings file on your filesystem and add ".js" to the file_extensions as in 
https://github.com/Monnoroch/ColorHighlighter/blob/master/ColorHighlighter.sublime-settings#L59

**Comment by enkia** [comment]
Looks like a lot of setting variables have changed. I deleted my user settings, started fresh, and it seems to be working for me. It doesn't highlight sass variables, though.
